### ISSUE TITLE
dpkg conflict with ipsw

### ISSUE THREAD

**Issue by arandomdev** [blacktop/go-apfs#26]
Trying to install this alongside with ipsw from the debs results in the following dpkg error,

```Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'go-apfs' instead of './go-apfs_1.0.16_linux_x86_64.deb'
The following NEW packages will be installed:
  go-apfs
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/2954 kB of archives.
After this operation, 6962 kB of additional disk space will be used.
Get:1 /home/haow6/go-apfs_1.0.16_linux_x86_64.deb go-apfs amd64 1.0.16 [2954 kB]
(Reading database ... 42651 files and directories currently installed.)
Preparing to unpack .../go-apfs_1.0.16_linux_x86_64.deb ...
Unpacking go-apfs (1.0.16) ...
dpkg: error processing archive /home/haow6/go-apfs_1.0.16_linux_x86_64.deb (--unpack):
 trying to overwrite '/usr/share/doc/ipsw/copyright', which is also in package ipsw 3.1.107
Errors were encountered while processing:
 /home/haow6/go-apfs_1.0.16_linux_x86_64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by blacktop** [comment]
@arandomdev thank you for letting me know! It was a copy-pasta error when I pulled all the CI/CD build stuff over from `ipsw` 👍 

It's building in Github Actions now and should be done shortly.

**Comment by blacktop** [comment]
I just tried this myself on an iOS16 SystemOS dmg and it's not working because it currently expects a FS dmg and the SystemOS is just a raw volume.

REF: #25

**Comment by arandomdev** [comment]
Good to know, thanks!

**Comment by arandomdev** [comment]
I'll go ahead and close this issue as it was fixed in [v1.0.17](https://github.com/blacktop/go-apfs/releases/tag/v1.0.17).
